README.md

```diff
@@ -46,7 +46,16 @@ faas-netes can be configured with environment variables, but for a full set of o
 | `httpProbe`         | Boolean - use http probe type for function readiness and liveness. Default: `false`             |
 | `write_timeout`     | HTTP timeout for writing a response body from your function (in seconds). Default: `60s`        |
 | `read_timeout`      | HTTP timeout for reading the payload from the client caller (in seconds). Default: `60s`        |
-| `image_pull_policy` | Image pull policy for deployed functions (`Always`, `IfNotPresent`, `Never`.  Default: `Always` |
+| `image_pull_policy` | Image pull policy for deployed functions (`Always`, `IfNotPresent`, `Never`).  Default: `Always` |
+| `gateway.resource`        | CPU/Memory resource requests/limits (memory: `120Mi`, cpu: `50m`) |
```

Comment:
Should this be `resource` or `resources`?

Reply:
Pretty sure it should be `resources` looking at the Kubernetes YAML

Reply:
you are right, i misspelled it here https://github.com/openfaas/faas-netes/pull/462#issuecomment-507835566

Reply:
Changed resource to resource